linode/manager
# refactor: [M3-8785] - Remove feature flag, tracking events used for A/B testing in API CLI tools Modal

## Description 📝
This PR removes LD tracking events and feature flag used for collecting metrics for A/B testing in DX Tools Modal.

## Changes  🔄
List any change relevant to the reviewer.
- ...
- ...

## Target release date 🗓️
Please specify a release date to guarantee timely review of this PR. If exact date is not known, please approximate and update it as needed.


### Verification steps
(How to verify changes)
- Navigate to Linode create flow
- Verify button language to "View Code Snippets"
- Verify no regression in DX tools modal, code snippets, copy icon etc. 

## As an Author I have considered 🤔

*Check all that apply*
